ReactiveCocoa/Swift/Signal.swift

```diff
@@ -551,10 +551,27 @@ public func takeLast<T,E>(count: Int)(signal: Signal<T,E>) -> Signal<T,E> {
 	}
 }
 
+/// Forwards any values from `signal` until `predicate` returns false,
+/// at which point the returned signal will complete.
+public func takeWhile<T,E>(predicate: T -> Bool)(signal: Signal<T,E>) -> Signal<T,E> {
```

Comment:
To be clear, I'm saying it doesn't state one way or the other. But currently, mostly there's a space between generic type parameters.